lib/compiler-llvm/src/trampoline/wasm.rs

```diff
@@ -63,9 +63,9 @@ impl FuncTrampoline {
         );
 
         let trampoline_func = module.add_function(name, trampoline_ty, Some(Linkage::External));
-        trampoline_func
-            .as_global_value()
-            .set_section(FUNCTION_SECTION);
+        /*trampoline_func
+        .as_global_value()
+        .set_section(FUNCTION_SECTION);*/
```

Comment:
Is this intended?

Reply:
Yeah, I needed this to get the symbols to show up with objdump... it looks like it still works if I uncomment it though